prow/config/presubmits.yaml

```diff
@@ -949,6 +949,70 @@ presubmits:
           name: cgroup
         - emptyDir: {}
           name: varlibdocker
+  - name: istio-integration-servicemesh-2.4-mc
+    trigger: (?m)^/test( | .* )integration\-maistra\-servicemesh\-mc,?($|\s.*)
+    rerun_command: /test integration-maistra-servicemesh-mc
+    skip_report: false
+    max_concurrency: 2
+    always_run: true
+    branches:
+    - ^maistra-2.4$
+    decorate: true
+    path_alias: istio.io/istio
+    spec:
+      containers:
+      - command:
+        - entrypoint
+        - prow/integ-suite-kind.sh
+        - --topology
+        - MULTICLUSTER
+        - test.integration.servicemesh.kube
+        env:
+        - name: GOFLAGS
+          value: -mod=vendor
+        - name: BUILD_WITH_CONTAINER
+          value: "0"
+        - name: INTEGRATION_TEST_FLAGS
+          value: ' --istio.test.retries=1 '
+        # FIXME: Uncomment this when we have our proxy in place
+        # - name: ISTIO_ENVOY_BASE_URL
+        #   value: https://storage.googleapis.com/maistra-prow-testing/proxy
+        - name: TEST_SELECT
+          value: -postsubmit,-flaky,+multicluster
+
+        image: "quay.io/maistra-dev/maistra-builder:2.4"
+        imagePullPolicy: Always
+        securityContext:
+          privileged: true
+        volumeMounts:
+        - mountPath: /lib/modules
+          name: modules
+          readOnly: true
+        - mountPath: /sys/fs/cgroup
+          name: cgroup
+          readOnly: true
+        - mountPath: /var/lib/docker
+          name: varlibdocker
+          readOnly: false
+        resources:
+          limits:
+            memory: 24Gi
+            cpu: "4"
+          requests:
+            cpu: "4"
+            memory: 4Gi
```

Comment:
you think we can lower this like we did with the other 2.4 jobs?

Reply:
I was wondering about it as it will start more than 1 cluster. OTOH it's a request so if we leave the limits as they are it "should be" increased if needed.

Reply:
yeah, the reason we lowered this value in the other jobs was to speed up the execution of the jobs. If we request 4 cpus, this means that in our current cluster we can't afford two jobs at the same time. Which means more delay in the queue.